File: 프로그래머스/unrated/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.py
def solution(arr):
    answer = []
    for i in arr:
        if i%2 ==0 and i >= 50:
            answer.append(i//2)  
        elif i%2 != 0 and i<50:
            answer.append(i*2)
        else:
            answer.append(i)
    return answer
                
        
            
    

File: 프로그래머스/0/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr) {
        return Arrays.stream(arr).map(v -> {
            if(v >= 50 && v % 2 == 0) {
                return v / 2;
            }
            else if(v < 50 && v % 2 == 1) {
                return v * 2;
            }
            return v;
        }).toArray();
    }
}